.github/dependabot.yml

```diff
@@ -4,3 +4,4 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+  open-pull-requests-limit: 10
```

Comment:
⛔❓

Reply:
Sorry, I don't follow?

Reply:
Oh, it only displays it in the main "Files changed" view, that's weird. I was referring to the indicator that this file now doesn't end in a newline.

Reply:
![Screen Shot 2021-07-27 at 5 00 03 PM](https://user-images.githubusercontent.com/394835/127226665-10366c49-cc41-416e-9246-119dda53c5c8.png)

Reply:
Ah. Missed that. Thanks.